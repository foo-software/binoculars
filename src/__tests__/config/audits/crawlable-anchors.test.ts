import auditCrawlableAnchors from '../../../config/audits/crawlable-anchors';

describe('crawlable-anchors audit', () => {
  it('should match snapshot', () => {
    expect(auditCrawlableAnchors('en')).toMatchSnapshot();
  });
});
