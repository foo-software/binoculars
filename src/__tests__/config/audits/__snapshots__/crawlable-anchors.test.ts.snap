// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`crawlable-anchors audit should match snapshot 1`] = `
Object {
  "description": "Search engines may use \`href\` attributes on links to crawl websites. Ensure that the \`href\` attribute of anchor elements links to an appropriate destination, so more pages of the site can be discovered. [Learn More](https://support.google.com/webmasters/answer/9112205)",
  "title": "Links are crawlable",
}
`;
